### PR TITLE
Add test for Theano

### PR DESCRIPTION
**Pull request by mrodozov**
Add test for Theano to test the updated version on cmsdist (to 0.9.0)

### CODE REVIEW

**Comment by mrodozov** [comment]
please test with cms-sw/cmsdist#3492

File: PhysicsTools/PythonAnalysis/test/testTheano.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import theano
+from keras.models import Sequential
+from keras.layers import MaxPooling2D, Conv2D
+
+print keras.__version__
+print theano.__version__
+
+model = Sequential()
+model.add(Conv2D(32, 10, 10,border_mode='valid',input_shape = (1,100,100)))
+model.add(MaxPooling2D(pool_size=(2, 2))) # <-- crashes here after 9_2_4
```
Comment (smuzaffar) [inline PhysicsTools/PythonAnalysis/test/testTheano.py]: @mrodozov , you can remove this `# <-- crashes here after 9_2_4` comment

**Comment by davidlange6** [comment]
please test

**Comment by smuzaffar** [comment]
please test with cms-sw/cmsdist#3492

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
@davidlange6 , this PR just adds an extra unit test for Theano, so I am merging it for 94X.

**Pull request merged**